### PR TITLE
moved "Upload Hours" button from avatar menu to ADMIN menu

### PR DESCRIPTION
**Pull request by mvolkmann**

### CODE REVIEW

File: web-ui/src/components/menu/Menu.jsx [hunk 1]
```diff
@@ -380,6 +380,15 @@ function Menu() {
             </ListItem>
             <Collapse in={adminOpen} timeout="auto" unmountOnExit>
               {createListJsx(adminLinks, true)}
+              {isAdmin && (
```
Comment (mvolkmann) [inline web-ui/src/components/menu/Menu.jsx:383]: This code was moved here from below.
I had to customize the styling of the button to match the look of the links in the ADMIN menu.

File: web-ui/src/components/menu/Menu.jsx [hunk 2]
```diff
@@ -461,16 +470,6 @@ function Menu() {
             >
               Profile
             </MenuItem>
-            {isAdmin && (
-              <MenuItem
-                onClick={() => {
-                  closeAvatarMenu();
```
Comment (mvolkmann) [inline web-ui/src/components/menu/Menu.jsx:467]: I didn't need the call to `closeAvatarMenu` above since the button is no longer in that menu.

**Review by S78901 — APPROVED** [review]

File: web-ui/src/components/menu/Menu.jsx (hunk 1 above)
Comment (mkimberlin) [reply web-ui/src/components/menu/Menu.jsx:383]: Any reason not to just make it a `<ListItem>` like the others?

**Review by vhscom — APPROVED** [review]
Looks good to me. I see `button` prop on `ListItem` is deprecated.

File: web-ui/src/components/menu/Menu.jsx (hunk 1 above)
Comment (mvolkmann) [reply web-ui/src/components/menu/Menu.jsx:383]: I didn't realize I could attach an `onClick` to a `ListItem`. I pushed that change.

**Comment by mvolkmann** [comment]
> Looks good to me. I see `button` prop on `ListItem` is deprecated.

I removed those deprecated props.

**Review by mkimberlin — DISMISSED** [review]

**Review by mkimberlin — DISMISSED** [review]

**Review by mkimberlin — APPROVED** [review]

**Review by jackkeller — APPROVED** [review]
:shipit: Approved

**Pull request merged**